packages/augur-ui/src/modules/common/labels.styles.less

```diff
@@ -704,9 +704,11 @@ div.LinearPropertyViewTransaction {
 }
 
 .BulkTxLabel {
-  .mono-9;
+  margin: 0 @size-24 @size-24;
```

Comment:
how does the font size get set? don't want to inherit it.

Reply:
Via `DismissableNotice` css.

`.mono-9` was the old font for when there was just text, without the white/blue label

Reply:
ok, I rem now, works for me.